server/models/event.js

```diff
@@ -1,10 +1,14 @@
 const bookshelf = require('../database');
+const moment = require('moment');
 
 const Event = bookshelf.Model.extend({
   tableName: 'event',
   projects() {
     return this.hasMany('Event');
   },
+  formattedDate() {
+    return moment(this.attributes.date).format('dddd [the] Do [of] MMMM');
```

Comment:
Doesn't that implies local time @DanielBrierton ?

Reply:
Doesn't display time, so yeah, I think this is fine.

Reply:
Add tz